app/controllers/provider_interface/notes_controller.rb

```diff
@@ -0,0 +1,41 @@
+module ProviderInterface
+  class NotesController < ProviderInterfaceController
+    before_action :set_application_choice
+
+    def index
+      @provider_can_respond = current_provider_user.authorisation.can_make_decisions?(
+        application_choice: @application_choice,
+        course_option_id: @application_choice.offered_option.id,
+      )
+
+      @notes = @application_choice.notes.order('created_at DESC')
+    end
+
+    def show
+      @note = @application_choice.notes.find(params[:id])
+    end
+
+    def new
+      @new_note_form = ProviderInterface::NewNoteForm.new
+    end
+
+    def create
+      @new_note_form = ProviderInterface::NewNoteForm.new new_note_params
+
+      if @new_note_form.save
+        flash[:success] = 'Note successfully added'
+        redirect_to provider_interface_notes_path(@application_choice)
+      else
+        render(action: :new_note)
+      end
+    end
+
+  private
+
+    def new_note_params
```

Comment:
Could this be renamed to `note_params` so it's consistent with the param methods of other controllers.